item.html

```diff
@@ -2,9 +2,85 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>$Title$</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes"/>
+  <title>PZ_1_4</title>
+  <link rel="stylesheet" type="text/css" href="./css/item.css">
 </head>
 <body>
-$END$
+<div class="wrapper">
+  <header class="header">
+    <div class="header_menu" id="backMainPage">
+      <img src="/img/item/back.svg" alt="">
+    </div>
+    <div class="header_title">
+      <h3 class="title">Тур вихідного дня “Чор...</h3>
+    </div>
+  </header>
+  <main class="main">
+    <div class="card">
+      <img src="img/1.png" alt="">
```

Comment:
Please use relative path for avoiding problem this probkem

Reply:
<img width="687" alt="Screenshot 2023-09-10 at 17 18 10" src="https://github.com/web-202/web-pz-1-4/assets/10829855/e52af286-5e6b-4023-b437-80cb15b4c026">